frontend-ssr/pages/api/user/register.js

```diff
@@ -0,0 +1,56 @@
+//Imports 
+
+const User = require("../../../models/User.schema");
+import dbConnect from "../../../utils/dbConnect";
+const bcrypt = require("bcryptjs")
+dbConnect();
```

Comment:
I see you're not running `dbConnect` from `login.js`. Is this intended or you forgot about it?

Reply:
I forgot about it, resolve